pacai/bin/capture.py

```diff
@@ -610,8 +608,8 @@ def readCommand(argv):
     args['redTeamName'] = options.red
     args['blueTeamName'] = options.blue
 
-    if options.fixRandomSeed:
-        random.seed(FIXED_SEED)
+    if options.seed:
```

Comment:
Pacman has a comment here, but capture does not.
Make sure they are consistent.